cmd/fuzztool/fuzztool.go

```diff
@@ -187,21 +186,20 @@ func copyCorpus(srcRoot, dstRoot string) {
 	}
 }
 
-func main() {
-	debugF := flag.Bool("debug", false, "enable debug mode")
-	srcF := flag.String("src", "", "source, one of: 'seed', 'generated', or collected corpus' directory")
-	dstF := flag.String("dst", "", "destination, one of: 'seed', 'generated', or collected corpus' directory")
-	flag.Parse()
+var CLI struct {
```

Comment:
This variable needs to be documented. I'd say documentation should answer the question "why do we need it". 

Based on what I see in the Taskfile, I guess it's called `CLI` because we don't use flags anymore (but rather parse the command line), right? Is there a particular reason why this variable is exported?